flake.nix

```diff
@@ -0,0 +1,24 @@
+{
+  description = "A prisma test project";
```

Comment:
This could be something more specific to Iron Fish, since I think this will be an Iron Fish flake

Reply:
I forgot to change that last night 😔 I'll fix it